solutions/python-igraph/visualize_cliques/visualize_cliques.py

```diff
@@ -0,0 +1,12 @@
+import igraph as ig
+import matplotlib.pyplot as plt
+
+g = ig.Graph.Famous('Zachary')
+g.es['width'] = 0.5
```

Comment:
I think it would make more sense to provide this directly in the plotting, i.e. `ig.plot(..., edge_width=0.5)`, since it's a constant anyway.